### DISCUSSION TITLE
Birds: A ToonSquid Animation Tutorial

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#565] [Show and tell]
[Birds: A ToonSquid Animation Tutorial](https://youtu.be/KA3LYh_aqNw?si=a_fga2qpUVmzBjB_) To create this animation, I used some hacks that proved useful for what I was trying to achieve. What are those hacks, you'll soon find out when you watch this video.

**Comment by Cat-Butler** [comment]
Nice quick tutorial with good use of ToonSquid's mix of traditional-derived and digital features.

Interesting choice of a shape as the hierarchy parent, which works well in this instance. My first thought would be a translation layer, so this has given me something to think about for future animations - thanks!.

**Reply by DWrightAnimation** [reply]
I chose to do a drawing layer as the parent because the transformation layer doesn't let you control the length of the transformation, whereas with this, I can do a keyframe for the drawing layer, and one the next frame is blank, the child layer will snap back to its original position, prior to parenting.

**Reply by Cat-Butler** [reply]
Great point! 
Thank you for replying with that extra detail.
